cypress/e2e/cloud/usage.test.ts

```diff
@@ -2,7 +2,7 @@ import {Organization} from '../../../src/types'
 
 const statHeaders = [
   'Data In (MB)',
-  'Query Count',
+  'Query Count (Flux Only) ?',
```

Comment:
I just noticed the `?` here - doesn't the tooltip element cover that?

Reply:
This is a change I identified after running the ui integration tests, which failed without the change. This is part of a test that designates what text to look for in the header. `QuestionMarkToolip` is a Clockface component that, among other things, renders and formats a `?` as text wherever the component is rendered. As a result, the test needed to be updated to address the fact that the text within the header now includes that ?.

Reply:
Ah, that makes sense, thanks for the explanation!